code/datums/outfits/jobs/ghostroles.dm

```diff
@@ -110,11 +116,9 @@
 	id_type =  /obj/item/card/id/syndicate/ert/aa/guild_ert
 
 /decl/hierarchy/outfit/lss_ert
-	name = "Auditor"
-	l_ear = /obj/item/device/radio/headset/headset_com
-	head = /obj/item/clothing/head/helmet/technomancersuit //Looks good enuff and has good armor I guess
+	name = "Internal Affairs Agent"
+	l_ear = /obj/item/device/radio/headset/heads/merchant
 	uniform = /obj/item/clothing/under/suit_jacket/executive
-	suit = /obj/item/clothing/suit/storage/cargovest
```

Comment:
why no suit?

Reply:
Because they're not combatants, they're businessmen, and the vest clashes with the rest of the look of a corporate bigwig.